### PR TITLE
Integrated REVERSE_V2 operator from TFLite to TFLM

### PR DESCRIPTION
**Pull request by omkarak99**

### CODE REVIEW

File: tensorflow/lite/micro/kernels/reverse.cc [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+}
+
+TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
+  //const TfLiteTensor* input;
+  //TF_LITE_ENSURE_OK(context, GetInputSafe(context, node, kInputTensor, &input));
+  //const TfLiteTensor* axis_tensor;
+  //TF_LITE_ENSURE_OK(context,
+  //                  GetInputSafe(context, node, kAxisTensor, &axis_tensor));
+  const TfLiteEvalTensor* input =
+      tflite::micro::GetEvalInput(context, node, 0);
```
Comment (cad-audio) [inline tensorflow/lite/micro/kernels/reverse.cc:101]: Use kInputTensor enum instead of 0

File: tensorflow/lite/micro/kernels/reverse.cc [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  const TfLiteEvalTensor* axis_tensor =
+      tflite::micro::GetEvalInput(context, node, 1);
```
Comment (cad-audio) [inline tensorflow/lite/micro/kernels/reverse.cc:103]: Used kAxisTensor instead of 1

File: tensorflow/lite/micro/kernels/reverse.cc [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  if (!is_contiguous) {
+    TF_LITE_KERNEL_LOG(context, "Non-contiguous `axes` not supported");
+    return kTfLiteError;
+  }
+
+  //TfLiteTensor* output;
+  //TF_LITE_ENSURE_OK(context,
+  //                  GetOutputSafe(context, node, kOutputTensor, &output));
+  TfLiteEvalTensor* output =
+      tflite::micro::GetEvalOutput(context, node, 0);
```
Comment (cad-audio) [inline tensorflow/lite/micro/kernels/reverse.cc:139]: Use kOutputTensor instead of 0

File: tensorflow/lite/micro/kernels/reverse.cc (hunk 1 above)
Comment (omkarak99) [reply tensorflow/lite/micro/kernels/reverse.cc]: Change incorporated

File: tensorflow/lite/micro/kernels/reverse.cc (hunk 2 above)
Comment (omkarak99) [reply tensorflow/lite/micro/kernels/reverse.cc]: Change incorporated

File: tensorflow/lite/micro/kernels/reverse.cc (hunk 3 above)
Comment (omkarak99) [reply tensorflow/lite/micro/kernels/reverse.cc]: Change incorporated

**Pull request merged**